.azure-pipelines/run-test-scheduler-template.yml

```diff
@@ -92,15 +92,15 @@ parameters:
 
 - name: DUMP_KVM_IF_FAIL
   type: string
-  default: ""
+  default: True
```

Comment:
Set default to `true` and update type to `boolean`?

Reply:
Different types have different behaviors, in shell, it's true/false, but in Python, it's True/False.
To avoid including unnecessary logic in the parameter, I prefer to only use the 'string' type in the template and handle the type conversion in the Python scripts.

Reply:
Then it would be better to wrap the text using double quote like `"True"`. What's more, use `values` to limit the choices to just `"True"` and `"False"`. Parameters are arguments of this template. It's like exposing an API for this template. It is better to have more explicity.